### DISCUSSION TITLE
Support for linking same IDP in same Organisation for multiple domains

### DISCUSSION THREAD

**Discussion by the-green-man14** [keycloak/keycloak#34731] [Feedback]
This is sort of similar to:
https://github.com/keycloak/keycloak/discussions/32111

However, often customers have different domain names such as .com, .de, .uk etc however they all the same identity provider for these. 

At the moment, you can only add one domain in an organisation to one IDP. It would be great to be able to link multiple domains names to one idp in an organisation.

**Comment by ps92121** [comment]
yes, we have the same issue - we have organizations that have multiple email address domains, that we need to link to an identity provider...    there needs to be multi-select or ALL option.

**Comment by michael-chambers** [comment]
Same issue here. The only solution I see at the moment is to have multiple copies of the identity provider, link them all to the organization, and map each to a different domain. Then, of course, there are multiple AssertionConsumerService endpoints that need to be communicated back to the IdP. It seems like this could be more of a checkbox exercise.

**Comment by wschomburg** [comment]
Hi, have you seen https://github.com/keycloak/keycloak/issues/33804?
